BUILD

```diff
@@ -0,0 +1,35 @@
+load("@rules_java//java:defs.bzl", "java_binary", "java_lite_proto_library", "java_proto_library")
+load("@rules_proto//proto:defs.bzl", "proto_library")
+
+proto_library(
+  name = "incomplete_example_proto",
+  srcs = ["java/com/google/location/lbs/activity/audioset/dataprocessing/incomplete-example.proto"],
+  deps = ["@com_google_protobuf//:timestamp_proto"],
+)
+
+java_proto_library(
+  name = "incomplete_example_java_proto",
+  deps = [":incomplete_example_proto"],
+)
+
+java_binary(
+  name = "csv_parser_java",
+  srcs = ["java/com/google/location/lbs/activity/audioset/dataprocessing/CsvParser.java"],
+  main_class = "CsvParser",
```

Comment:
Once converted to java_library, we don't need the main_class specification.

Reply:
Ok. Thanks. I'll remove them.